.github/workflows/release.yml

```diff
@@ -16,12 +16,12 @@ jobs:
             name: linux
             binary_path: target/x86_64-unknown-linux-musl/release
             binary_files: icx
-            rust: 1.50.0 # gmiam/rust-musl-action@master is only at 1.50.0 now
+            rust: '1.58.1'
```

Comment:
Would you please test this workflow before merging?  To do so, you'd add the branch name from this PR to `push: branches: ` above.

Reply:
Works great

Reply:
Thanks!